package.json

```diff
@@ -21,8 +21,6 @@
     "fix": "lg-lint --fix",
     "link": "lg-link",
     "lint": "lg-lint",
-    "prepublishOnly": "yarn fix && yarn test && yarn build && yarn prepublish:ts-downlevel",
-    "prepublish:ts-downlevel": "rm -rf packages/*/dist/ts3.4 && echo packages/*/dist | NODE_NO_WARNINGS=1 xargs -I{} -n1 -P8 downlevel-dts {} {}/ts3.4",
     "publish": "yarn changeset publish --public",
```

Comment:
do we need to explicitly build pre publish?

Reply:
Ah, that's a good point